README.md

```diff
@@ -53,6 +53,85 @@ pip install -r requirements.txt -r requirements_dev.txt
 ```
 This script runs pytest on the `./tests` directory.
 
+## Debugging
+
+It is possible to debug this service for development by attaching a debugger
+that follows the
+[debugger-adapter-protocol](https://microsoft.github.io/debug-adapter-protocol/)
+to a harmony stack running in a local kubernetes cluster.  These instructions
+are for developers of this service in order to help them understand the code.
+
+Basic steps for debugging are:
+
+1. Add [debugpy](https://github.com/microsoft/debugpy) to `requirements.txt` file and reubild this image `./bin/build-image`.
+     - Generally this would just be adding a line to the end of the file that
+       says `debugpy==1.6.0` But there is a bug in that version and you need to
+       install from github directly until debugpy > 1.6 exists.  So for now,
+       you must install a specific version of `debugpy` and add `git` to the
+       service.Dockerfile
```

Comment:
I took a quick look at the issues in the `debugpy` repository, but wasn't sure which one corresponded to the issue you were referring to here. Is it also possible to pin the version to an older one, like `debugpy==1.5.1` or something, to still install via Pip from PyPI via `requirements.txt`?

Reply:
I didn't think to go back and see if it was a regression.  Will definitely try that. I can find the bug report and link it in the PR.

Reply:
I tracked down the problem on the [emacs side](https://github.com/emacs-lsp/dap-mode/issues/625#issuecomment-1141082447). This is the fix in [debugpy](https://github.com/microsoft/debugpy/pull/928).  I just tried with v1.5.1 and that displays the same error.